tests/generate_acrastruct.py

```diff
@@ -19,10 +19,17 @@
 parser.add_argument(
     '--data', nargs='?', default='Plain text.',
     help='Plain text to encode (default: "Plain text.")')
+parser.add_argument(
+    '--out_file', nargs='?', default='',
+    help='Save AcraStruct to filename (default: "<client_id>.acrastruct")')
 args = parser.parse_args()
 
+as_out_file = args.out_file
+if as_out_file == '':
+    as_out_file = '{}.acrastruct'.format(args.client_id)
```

Comment:
example how we can avoid extra if
```
default_client_id = "testclientid"
parser.add_argument(
    '--client_id', nargs='?', default=default_client_id,
    help='Client ID (default: testclientid)')
parser.add_argument(
    '--out_file', nargs='?', default='{}.acrastruct'.format(args.client_id),
    help='Save AcraStruct to filename (default: "<client_id>.acrastruct")')
```
but in general no matter)

Reply:
Many thanks! But I still prefer the existent code as more clear.